Swift code guidelines.md

```diff
@@ -26,7 +26,22 @@ func handle(response: Response)
 2.2. Start factory methods with word `make`.
 2.3. Choose clarity over brevity. 
 2.4. Name booleans like `isAuthenticated` instead of `authenticated`. 
-2.5. Name function parameters. 
+2.5. Name function parameters where it is appropriate. 
+
+**Preferred:**
+```swift
+func move(from start: Point, to end: Point)
+func makeRequest(withTimeout timeout: TimeInterval, headers: [String: String])
```

Comment:
This is the same as the equivalent one in the Not Preferred section

Reply:
You have an eagle eye :) Thanks! 
I fixed that.